src/components/sbb-time-input/sbb-time-input.scss

```diff
@@ -0,0 +1,24 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-time-input-max-width: #{sbb.px-to-rem-build(58)};
+
+  @include sbb.mq($from: medium) {
+    --sbb-time-input-max-width: #{sbb.px-to-rem-build(65)};
+  }
+
+  display: inline-block;
+}
+
+input {
+  @include sbb.input-reset;
+
+  max-width: var(--sbb-time-input-max-width);
+  font-size: var(--sbb-font-size-text-m);
+  line-height: var(--sbb-typo-line-height-body-text);
+  color: var(--sbb-form-field-text-color);
```

Comment:
This only works if used with sbb-form-field, but maybe also the desired behavior. If so, I suggest to just add a comment.

Reply:
Yes, the color rule doesn't work, e.g. the disabled state is not shown with gray text if the component is NOT used within a sbb-form-field. https://63ca6f1f40bf1c0084a928a7--lyne-components-storybook.netlify.app/?path=/story/components-form-elements-sbb-time-input--sbb-time-input&args=disabled:true

@kyubisation Are there use cases with the time-input without form-field? If yes as i think, maybe I have to add here the rules for the color handling..

Reply:
We can ignore the "standalone" variant for now. Might make sense to also remove the story for it.